src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return 80;
```

Comment:
The return value here should be based on a calculation involving the number of days and the daily rate, not a hardcoded value. The daily rate should be a constant, and the total cost should be calculated as `days * DAILY_RATE`.